var/spack/repos/builtin/packages/dnstop/package.py

```diff
@@ -15,6 +15,10 @@ class Dnstop(AutotoolsPackage):
     version('master', branch='master')
 
     depends_on('libpcap')
+    depends_on('ncurses')
+
+    def configure_args(self):
+        return ['LIBS=-ltinfo']
```

Comment:
I think this is not entirely correct, ncurses has a variant `termlib` which influence whether or not libtinfo exists iirc, can you verify?